swc/private/is_bazel_7_or_greater.bzl

```diff
@@ -0,0 +1,7 @@
+"is_bazel_7_or_greater"
+
+def is_bazel_7_or_greater():
```

Comment:
This will be removed shortly on the bump to rules_js 2.0 when we also bump the min aspect-bazel-lib version to a version that has this function